test/test_google_ads_client.rb

```diff
@@ -228,4 +228,21 @@ def test_authentication_overrides_updater_proc_and_keyfile
     credentials = client.get_credentials
     assert_equal('path/to/file', credentials)
   end
+
+  def test_load_environment_variables
+    # Test config file path
+    ENV.store("GOOGLE_ADS_CONFIGURATION_FILE_PATH", "test/fixtures/config.rb")
+    client = Google::Ads::GoogleAds::GoogleAdsClient.new
+    assert_equal(client.config.developer_token, "INSERT_DEVELOPER_TOKEN_HERE")
+    assert_equal(client.config.client_id, "INSERT_CLIENT_ID_HERE")
+    assert_equal(client.config.client_secret, "INSERT_CLIENT_SECRET_HERE")
+    # Test environment variable overrides
+    ENV.store("GOOGLE_ADS_DEVELOPER_TOKEN", "XYZ")
```

Comment:
Move these to the top so that we can test they're not loaded until the user manually calls the method as well.

Reply:
Done.